src/matrix_utils/matrix_funcs.py

```diff
@@ -0,0 +1,29 @@
+from matrix_utils import matrix_classes
+# PyCharm ругает: Unresolved reference 'matrix_utils' и 'matrix_classes', но без 1стр не работает
```

Comment:
1 вариант. Попробуй так

```suggestion
from src.matrix_utils import matrix_classes
```

2. Вариант (советую): 
В пайчарме выбрать папку src и сделать ее source
<img width="397" alt="Снимок экрана 2021-08-02 в 11 19 45 AM" src="https://user-images.githubusercontent.com/35404746/127828135-7b58680a-ced1-4b05-9e23-214bc8ddf27b.png">

Reply:
Помогло, спасибо!